src/Orleans.Core/Configuration/Options/LoadSheddingOptions.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+
+namespace Orleans.Hosting
+{
+    public class LoadSheddingOptions
+    {
+        /// <summary>
+        /// The LoadShedding element specifies the gateway load shedding configuration for the node.
+        /// If it does not appear, gateway load shedding is disabled.
+        /// </summary>
+        public bool LoadSheddingEnabled { get; set; }
+
+        /// <summary>
+        /// The LoadLimit attribute specifies the system load, in CPU%, at which load begins to be shed.
+        /// Note that this value is in %, so valid values range from 1 to 100, and a reasonable value is
+        /// typically between 80 and 95.
+        /// This value is ignored if load shedding is disabled, which is the default.
+        /// If load shedding is enabled and this attribute does not appear, then the default limit is 95%.
+        /// </summary>
+        public int LoadSheddingLimit { get; set; } = DEFAULT_LOAD_SHEDDING_LIMIT;
+        public const int DEFAULT_LOAD_SHEDDING_LIMIT = 95;
+    }
+
+    public class LoadSheddingOptionsFormatter : IOptionFormatter<LoadSheddingOptions>
+    {
+        public string Category { get; }
+
+        public string Name => nameof(LoadSheddingOptions);
```

Comment:
I feel that we've swapped `Category` and `Name` in almost every IOptionFormatter instance.

`Category` = the option class name
`Name` = the named options instance (i.e, for named options)

@jason-bragg @xiazen am I wrong about this?

Reply:
Category means the category this component is in. For example, AzureTableStorageOption should have category name Storage, and name AzureTableStorageOption-name

If Category is too confusing, even for us, maybe we should just remove it.

Reply:
@xiazen I agree with removing it. We can use `Name` values like `Storage.AzureTableStorageOptions.Default` and `Storage.AzureTableStorageOptions.MyHappyLittleGrainStore` to deal with named options and categories of options. That way, we can sort all formatters lexicographically when we display them and they'll be grouped nicely.

Reply:
sounds good. I will have a PR out for removing it. or @jason-bragg do you want to do it? since you are touching formatter a lot. I don't want to produce merge conflicts for you